views/forms/intake_packet.php

```diff
@@ -0,0 +1,1109 @@
+                                      formofchildhoodabuse := $31::TEXT,
+                                      hashadtherapy := $32::BOOLEAN,
+                                      stillissuesfromchildabuse := $33::BOOLEAN,
+                                      mostimportantliketolearn := $34::TEXT,
+                                      hasdomesticviolencehistory := $35::BOOLEAN,
+                                      hasdiscusseddomesticviolence := $36::BOOLEAN,
+                                      hashistorychildabuseoriginfam := $37::BOOLEAN,
+                                      hashistoryviolencenuclearfamily := $38::BOOLEAN,
+                                      ordersofprotectioninvolved := $39::BOOLEAN,
+                                      reasonforordersofprotection := $40::TEXT,
+                                      hasbeenarrested := $41::BOOLEAN,
+                                      hasbeenconvicted := $42::BOOLEAN,
+                                      reasonforarrestorconviction := $43::TEXT,
+                                      hasjailrecord := $44::BOOLEAN,
+                                      hasprisonrecord := $45::BOOLEAN,
+                                      offensejailprisonrec := $46::TEXT,
+                                      currentlyonparole := $47::BOOLEAN,
+                                      onparoleforwhatoffense := $48::TEXT,
+                                      ptpmainformsigneddate := $49::DATE,
+                                      ptpenrollmentsigneddate := $50::DATE,
+                                      ptpconstentreleaseformsigneddate := $51::DATE,
+                                      eID := $52::INT
+                                      );", [$pIDResult, $intake_dob, $intake_ethnicity, 'Male', $intake_street_num, $intake_street_name, $intake_intake_apt_info, $intake_zip, $intake_city, $intake_state,
+                                            $intake_occupation, $intake_religion, $intake_handicap_medication, $intake_last_year_school, $drug_alcohol_abuse, $drug_alcohol_abuse_explain, $live_with_children_separated,
+                                            $separated_length, $relationship, $parenting, $child_protective, $previous_child_protective, $mandated, $mandated_by, $reason_for_attendance, $class_participation, $parenting_opinion,
+                                            $other_classes, $other_classes_where_when, $victim_of_abuse, $form_of_abuse, $abuse_therapy, $childhood_abuse_relating, $class_takaway, $domestic_violence, $domestic_violence_discussed,
+                                            $history_violence_family, $history_violence_nuclear, $protection_order, $protection_order_explain, $crime_arrested, $crime_convicted, $crime_explain,
+                                            $jail_prison_record, TRUE, $jail_prison_explain, $parole_probation, $parole_probation_explain, $datestamp, $datestamp, $datestamp, $eID]);
```

Comment:
I'm gonna say the same thing here as I did with the referral form. You should validated that these queries actually ran properly since PHP will not throw an error if a query fails:

```php
if ($formID) {
    $state = pg_result_error_field($formID, PGSQL_DIAG_SQLSTATE);
    if ($state != 0) {
        die(pg_result_error($formID)); // change this to display an error notification instead
    }
}
```

I get the same issue with the zip codes when defaulting to 12601 and it hasn't been created yet in the `ZipCodes` table. (See my comment on #98)